maths/jaccard_similarity.py

```diff
@@ -67,14 +70,14 @@ def jaccard_similariy(setA, setB, alternativeUnion=False):
 
         if alternativeUnion:
             union = len(setA) + len(setB)
+            return len(intersection) / union
         else:
             union = setA + [element for element in setB if element not in setA]
-
-        return len(intersection) / len(union)
+            return len(intersection) / len(union)
 
```

Comment:
This is verified, right?

Reply:
No, I mean the tests do go through those branches. Earlier, there was just one return statement
```python
        return len(intersection) / len(union)
```
But, now the first return statement is different:
```python
            return len(intersection) / union
```